src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets

```diff
@@ -104,7 +110,14 @@ Copyright (c) .NET Foundation. All rights reserved.
       <UseSharedCompilation>true</UseSharedCompilation>
     </PropertyGroup>
 
-    <Csc
+    <PropertyGroup>
+      <LangVersion Condition="'$(LangVersion)' == '' AND
+        (('$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(TargetFrameworkVersion)' == 'v3.0') OR
+         ('$(TargetFrameworkIdentifier)' == '.NETStandard' AND '$(TargetFrameworkVersion)' == 'v2.1'))">preview</LangVersion>
```

Comment:
So one interesting issue. AspNetCore-Tooling's current sdk doesn't have a new enough Roslyn to understand "preview". I was under the assumption that Roslyn's master branch is targeting 16.1 hence why I based these changes off of that. I can either force this to C# 8 for now or make this file mimic what the latest SDK we've consumed has. 

@jcouv @jasonmalinowski @jinujoseph  What Roslyn branch should we be basing our changes off of for VS 16.1 preview 1? I couldn't find a 16.1 branch that stood out so I assumed master.